ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -247,7 +247,7 @@ StatusDialog {
             // in localized version. It should be refactored to provide raw
             // number consistently. Only the displaying component should apply
             // final localized formatting.
-            const delocalized = popup.preDefinedAmountToSend.replace(",", ".")
+            const delocalized = popup.preDefinedAmountToSend.replace(LocaleUtils.userInputLocale.decimalPoint, ".")
```

Comment:
What's the purpose of that chage? I would leave it as it was because it was simpler and even more resilient to potential problems with formatting of incoming string (e.g. with "," when locale decimal point is ".").

Reply:
Ok, I read @Seitseman comment. Indeed it's justified if we consider more options than only "." and ",". Anyways I hope it will be refactored and removed, as described in the comment.